src/main/java/org/ice4j/ice/harvest/MappingCandidateHarvesters.java

```diff
@@ -284,11 +285,18 @@ public StunMappingCandidateHarvester call()
         // Now run discover() on all created harvesters in parallel and pick
         // the ones which succeeded.
         ExecutorService es = Executors.newFixedThreadPool(tasks.size(),
-            runnable -> {
-                Thread thread = new Thread(runnable);
-                thread.setName("ICE Harvester Executor");
-                return thread;
-            });
+            new ThreadFactory() {
```

Comment:
You could use [CustomizableThreadFactory](https://github.com/jitsi/jitsi-utils/blob/master/src/main/java/org/jitsi/utils/concurrent/CustomizableThreadFactory.java)

Reply:
Actually even better it looks like we can use `ExecutorService#createFixedThreadPool` which itself uses `CustomizableThreadFactory` -- ice4j is already using the `ExecutorService` methods in a bunch of places.